helix-term/src/ui/mod.rs

```diff
@@ -288,8 +288,19 @@ pub mod completers {
             serde_json::json!(Config::default())
                 .as_object()
                 .unwrap()
-                .keys()
-                .cloned()
+                .iter()
+                .flat_map(|(key, value)| {
+                    if let Some(map) = value.as_object() {
+                        let mut v: Vec<String> = map
+                            .keys()
+                            .map(|sub_key| format!("{}.{}", key, sub_key))
```

Comment:
This only works one level deep though, right?

Reply:
That's correct. Fixed it now to work recursively for all keys no matter how deep.